Makefile

```diff
@@ -30,7 +30,7 @@ TECH_SUFFIX = .bgdi.ch
 API_URL ?= //api3.geo.admin.ch
 API_TECH_URL ?= //mf-chsdi3.
 LAST_API_URL := $(call lastvalue,api-url)
-ALTI_URL ?= //api3.geo.admin.ch
+ALTI_URL ?= //alti.geo.admin.ch
 LAST_ALTI_URL := $(call lastvalue,alti-url) 
 ALTI_TECH_URL ?= //mf-chsdi3.
```

Comment:
Just see it should be service-alti no?

Reply:
`alti.geo.admin.ch` or `service-alti.prod.bgdi.ch`

Reply:
So the ALTI_TECH_URL should be //service-alti.